Signal/src/ViewControllers/OWSConversationSettingsTableViewController.m

```diff
@@ -209,10 +211,30 @@ - (void)updateTableContents
                                          }]];
     }
 
+    [firstSectionItems addObject:[OWSTableItem itemWithCustomCellBlock:^{
```

Comment:
There are certain pieces of code I have rewritten from scratch every time I switch employers. This class is (part of) one of them.